test/Microsoft.AspNetCore.JsonPatch.Test/Dynamic/DynamicObjectAdapterTests.cs

```diff
@@ -0,0 +1,1215 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using Microsoft.AspNetCore.JsonPatch.Exceptions;
+using Newtonsoft.Json;
+using Xunit;
+
+namespace Microsoft.AspNetCore.JsonPatch.Test.Dynamic
+{
+    public class DynamicObjectAdapterTests
```

Comment:
I tried to be consistent with https://github.com/aspnet/JsonPatch/blob/dev/test/Microsoft.AspNetCore.JsonPatch.Test/Dynamic/AddOperationTests.cs and other tests [here](https://github.com/aspnet/JsonPatch/tree/dev/test/Microsoft.AspNetCore.JsonPatch.Test/Dynamic). Should I move my tests to the existing test classes? They are organized by `OperationType`, not by `Adapters`.